atsha204a/src/cmd.c

```diff
@@ -405,6 +407,19 @@ int cmd_get_slot_config(struct io_interface *ioif, uint8_t slotnbr,
 	return ret;
 }
 
+int cmd_update_extra(struct io_interface *ioif, uint8_t mode, uint8_t value)
+{
+	uint8_t resp_buf;
+	struct cmd_packet p;
+	int ret = STATUS_EXEC_ERROR;
+
+	get_command(&p, OPCODE_UPDATEEXTRA);
+	p.param1 = mode;
+	p.param2[0] = value;
+
+	ret = at204_msg(ioif, &p, &resp_buf, sizeof(resp_buf));
+}
```

Comment:
I saw this after merging. The function should return something (which it currently does not).
```c
return at204_msg(ioif, &p, &resp_buf, sizeof(resp_buf));
```
I suppose (I can fix this while merging others)